File: Frontend/Pizzeria/src/app/Shared/model/order.ts
export class Order {
    id: number;
    pizzaId: number;
    orderStatus: string;

    constructor(order) {
        {
            this.id = order.id;
            this.pizzaId = order.pizzaId;
            this.orderStatus = order.orderStatus || '';
        }
    }
}

File: Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TaviscaAssessment.Models
{
    public class Order : Entity
    {
        public string OrderStatus { get; set; }

        public int PizzaId { get; set; }
        public virtual Pizza Pizza { get; set; }
    }
}
